examples/pytorch/byol.py

```diff
@@ -0,0 +1,79 @@
+import torch
+from torch import nn
+import torchvision
+import copy
+
+import lightly
+from lightly.models.modules import BYOLProjectionHead
```

Comment:
I'll just write "consistent imports" everywhere now since it seems there's the same problem everywhere